twake/backend/node/src/cli/cmds/migration_cmds/php-message/message-migrator-service.ts

```diff
@@ -68,15 +68,23 @@ class MessageMigrator {
     let pageDirectChannels: Pagination = { limitStr: "100" };
     // For each directChannels find messages
     do {
-      const directChannelsInCompanyResult = await this.channelService.channels.getDirectChannelsInCompany(
+      const directChannelsInCompanyResult = await this.channelService.channels.list(
         pageDirectChannels,
-        company.id,
+        {},
+        {
+          workspace: {
+            workspace_id: 'direct',
+            company_id: workspace.group_id,
+          },
+          user: { id: null, server_request: true },
+        },
       );
+
```

Comment:
@stephanevieira75 take a look at this, this is what you needed to change to get direct channel the good way